.github/workflows/ci-gradle.yml

```diff
@@ -52,20 +69,27 @@ jobs:
         run: /bin/sh .github/fetch-robolectric-dependencies.sh
 
       # Build the entire project, run the tests, and run all static analysis
-      - name: Gradle Build
-        run: ./gradlew assembleRelease check -x embrace-gradle-plugin-integration-tests:test --stacktrace
+      - name: Gradle (embrace-gradle-plugin-integration-tests:test)
+        if: ${{ matrix.gradle-task == 'test' }}
+        run: ./gradlew embrace-gradle-plugin-integration-tests:test --stacktrace --parallel
```

Comment:
--parallel doesn't have an effect as we already have `org.gradle.parallel` in the gradle.properties

Reply:
yep, i was trying to check if gradle.props is definitely taking effect - which it does, as --parallel made no difference